skimage/restoration/tests/test_denoise.py

```diff
@@ -523,13 +542,15 @@ def test_wavelet_denoising_args():
 
     for convert2ycbcr in [True, False]:
         for multichannel in [True, False]:
+            expected = (PYWAVELET_ND_INDEXING if multichannel else None)
```

Comment:
I'd refrain from using name `expected`. Usually, it is assigned to an _expected result_ of the function-under-test.